frontend/e2e-test/test/e2e/dev-api/fixtures.ts

```diff
@@ -79,7 +79,7 @@ export const daycareFixture: Daycare = {
   id: '4f3a32f5-d1bd-4b8b-aa4e-4fd78b18354b',
   careAreaId: careAreaFixture.id,
   name: 'Alkuräjähdyksen päiväkoti',
-  type: ['CENTRE'],
+  type: ['CENTRE', 'PRESCHOOL'],
```

Comment:
Not sure if this is a great idea, but it avoids [this](https://github.com/espoon-voltti/evaka/blob/b88fc70c7c767211bd918fb40779ed60dc7edfb0/frontend/packages/employee-frontend/src/components/unit/unit-details/UnitEditor.tsx#L331) error.

Reply:
This is fine 👍